docs/src/userguide/cube_statistics.rst

```diff
@@ -23,9 +23,9 @@ Collapsing Entire Data Dimensions
 
 In the :doc:`subsetting_a_cube` section we saw how to extract a subset of a
 cube in order to reduce either its dimensionality or its resolution.
-Instead of simply extracting a sub-region of the data, 
-we can produce statistical functions of the data values 
-across a particular dimension, 
+Instead of simply extracting a sub-region of the data,
+we can produce statistical functions of the data values
+across a particular dimension,
```

Comment:
My IDE will fix any poorly formatted text in files that it touches, and unfortunately I accidentally touched some extra files and didn't catch it in time (4cf57f06256a11355494a24040d056e2e050aa93).

The spaces at ends of lines shouldn't be there; this is an improvement, just not one that was asked for or expected!

Reply:
These changes aren't associated with any unstructured work.  Please can you back them out as they are highly likely to cause conflicts when merging onto main later.

Reply:
Having discussed offline, we haven't been able to work out a way to avoid the potential conflict. If this happened again I would have spent the extra 15mins undoing the original mistake, to avoid this later problem.